sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -245,7 +246,7 @@ public EventParticipantDto getByUuid(String uuid) {
 		Integer max,
 		List<SortProperty> sortProperties) {
 
-		if (eventParticipantCriteria == null || eventParticipantCriteria.getEvent() == null) {
+		if ((eventParticipantCriteria == null || eventParticipantCriteria.getEvent() == null) && eventParticipantCriteria.getPerson() == null) {
```

Comment:
I think this will produce a NullPointerException when eventParticipantCriteria is null. Suggestion:

`if (eventParticipantCriteria == null || (eventParticipantCriteria.getEvent() == null && eventParticipantCriteria.getPerson() == null))`